### PR TITLE
National indicator tweaks

### PR DESCRIPTION
**Pull request by rogeringle**
Some minor edits to the BIP website homepage and indicator pages

### CODE REVIEW

File: app/views/cms/homepage/_indicators.html.erb [hunk 1]
```diff
@@ -14,18 +14,21 @@
...
-      <% end %>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/africa">Africa</a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/asia-pacific">Asia & Pacific </a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/europe">Europe</a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/latin-america-caribbean">Latin America & Caribbean </a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/north-america">North America</a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/polar">Polar</a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/africa">west-asia</a>
```
Comment (drewpotter) [inline app/views/cms/homepage/_indicators.html.erb]: West Asia is lower case unlike the rest of the links here.

Comment (drewpotter) [reply app/views/cms/homepage/_indicators.html.erb]: Also these links currently aren't working for me unless they are not supposed to link to something right now?

Comment (rogeringle) [reply app/views/cms/homepage/_indicators.html.erb]: Links are on the live site only, so should work.

The West Asia thing is me rushing, fixing now!

**Comment by rogeringle** [comment]
Ready to be looked at again @drewpotter

File: app/views/cms/homepage/_indicators.html.erb [hunk 2]
```diff
@@ -14,18 +14,21 @@
...
-        <% all_indicators.sort_by(&:label).each do |indicator| %>
-          <a href="<%= indicator.full_path %>" class="homepage-targets__target-indicator">
-            <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i><%= indicator.label %>
-        </a>
-        <% end %>
-      <% else %>
-        <p class="homepage-themes__theme-indicator homepage-themes__theme-indicator--bold">No Global Indicators available yet</p>
-      <% end %>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/africa">Africa</a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/asia-pacific">Asia & Pacific </a>
```
Comment (drewpotter) [inline app/views/cms/homepage/_indicators.html.erb]: I noticed some extra spaces in this after the name of the region.

File: app/views/cms/homepage/_indicators.html.erb [hunk 3]
```diff
@@ -14,18 +14,21 @@
...
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/europe">Europe</a>
+     <i class="homepage-themes__theme-indicator-icon fa fa-chevron-right"></i> <a href="/national-indicator-development/national-data/latin-america-caribbean">Latin America & Caribbean </a>
```
Comment (drewpotter) [inline app/views/cms/homepage/_indicators.html.erb]: Again I noticed some extra spaces in this after the name of the region.

Comment (rogeringle) [reply app/views/cms/homepage/_indicators.html.erb]: Sloppy me. Fixing now.

**Comment by rogeringle** [comment]
Spaces removed @drewpotter

**Comment by drewpotter** [comment]
Good job! I am going to merge this now.

**Pull request merged**